### PR TITLE
TMDM-11900  Avoid duplicate key in Update_Report tables

### PR DESCRIPTION
**Pull request by hw-zhu**
**What is the current behavior?** (You should also link to an open issue here)
there is duplicate key in Update_Report tables

**What is the new behavior?**
using UUID to avoid duplicate key in Update_report ables

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: org.talend.mdm.core.storage.sql/src/test/resources/com/amalto/core/query/TypeOrdering.xsd [hunk 1]
```diff
@@ -912,12 +912,12 @@
...
         <xsd:unique name="Update">
             <xsd:selector xpath="."/>
-            <xsd:field xpath="Source"/>
-            <xsd:field xpath="TimeInMillis"/>
+            <xsd:field xpath="ID"/>
         </xsd:unique>
     </xsd:element>
     <xsd:complexType abstract="false" mixed="false" name="Update">
         <xsd:sequence maxOccurs="1" minOccurs="1">
+        	<xsd:element maxOccurs="1" minOccurs="1" name="ID" nillable="false" type="xsd:string"/>
```
Comment (lpw82gf) [inline org.talend.mdm.core.storage.sql/src/test/resources/com/amalto/core/query/TypeOrdering.xsd]: Fix the indentation

File: org.talend.mdm.webapp.journal/src/test/java/org/talend/mdm/webapp/journal/server/service/JournalDBServiceTest.java [hunk 2]
```diff
@@ -193,7 +193,6 @@ public void testParseString2Model() throws NoSuchMethodException, InvocationTarg
         assertEquals("DStar", returnValue.getDataContainer());
         assertEquals("DStar", returnValue.getDataModel());
         assertEquals("Agency", returnValue.getEntity());
-        assertEquals("genericUI.1360032633336", returnValue.getIds());
```
Comment (lpw82gf) [inline org.talend.mdm.webapp.journal/src/test/java/org/talend/mdm/webapp/journal/server/service/JournalDBServiceTest.java]: Do we need to assert the `source` and `timeInMillis` separately?

File: org.talend.mdm.core.storage.sql/src/test/resources/com/amalto/core/query/TypeOrdering.xsd (hunk 1 above)
Comment (hw-zhu) [reply org.talend.mdm.core.storage.sql/src/test/resources/com/amalto/core/query/TypeOrdering.xsd]: have fixed

File: org.talend.mdm.webapp.journal/src/test/java/org/talend/mdm/webapp/journal/server/service/JournalDBServiceTest.java (hunk 2 above)
Comment (hw-zhu) [reply org.talend.mdm.webapp.journal/src/test/java/org/talend/mdm/webapp/journal/server/service/JournalDBServiceTest.java]: Assert source and timeInMillis separately have existed. I added a fixed UUID to check it now.

**Pull request merged**